src/ni/vsbuild/Pipeline.groovy

```diff
@@ -178,7 +178,7 @@ class Pipeline implements Serializable {
          script.stage('Checkout_readBuildInformation') {
             script.deleteDir()
             script.echo 'Attempting to get source from repo.'
-            script.timeout(time: 5, unit: 'MINUTES'){
+            script.timeout(time: 10, unit: 'MINUTES'){
```

Comment:
I'm fine with this change as-is for now, but if we need to increase this value in the future, we should consider just making it configurable on a per-repo basis (e.g. specified in the Jenkinsfile or build.toml file).